fastlane/Fastfile

```diff
@@ -777,13 +780,18 @@ lane :rubocop do
   sh('bundle exec rubocop')
 end
 
+desc 'Remove UI snapshots'
+private_lane :remove_snapshots do
+  Dir.glob("../Tests/StreamChatUITests/**/__Snapshots__/**/*.png").select { |file| File.delete(file) }
+end
```

Comment:
What is the goal of this? 🤔

Reply:
To remove all png files if argument `record` is provided. I found it easy to collect snapshots from cli using fastlane.